SwerveControllerCommand/robotcontainer.py

```diff
@@ -0,0 +1,114 @@
+"""
+Copyright (c) FIRST and other WPILib contributors.
+Open Source Software; you can modify and/or share it under the terms of
+the WPILib BSD license file in the root directory of this project.
+"""
+
+from wpilib import XboxController
+import commands2
+import constants as Constants
+import Subsystems.DriveSubsystem as DriveSubsystem
+from wpimath.trajectory import  Trajectory, TrajectoryConfig, TrajectoryGenerator
+from wpimath.geometry import Translation2d, Pose2d, Rotation2d
+from wpimath.controller import ProfiledPIDController,PIDController
+import math
+
+
+class RobotContainer:
+    def __init__(self):
+        """
+        The robot's subsystems
+        """
+        self.m_robotDrive = DriveSubsystem()
+
+        """
+        The driver's controller
+        """
+        self.m_driverController = XboxController(Constants.OIConstants.kDriverControllerPort)
+
+        """
+        Configure default commands
+        """
+        self.m_robotDrive.setDefaultCommand(
+            """
+            The left stick controls translation of the robot.
+            turning is controlled by the X axis of the right stick.
+            """
+            commands2.RunCommand(
+                lambda: self.m_robotDrive.drive(
+                    """
+                    Multiply by max speed to map the joystick unitless inputs to actual units.
+                    This will map the [-1, 1] to [max speed backwards, max speed forwards],
+                    converting them to actual units.
+                    """
+                    self.m_driverController.getYChannel() * Constants.DriveConstants.kMaxSpeedMetersPerSecond,
+                    self.m_driverController.getXChannel() * Constants.DriveConstants.kMaxSpeedMetersPerSecond,
+                    self.m_driverController.getX(Constants.Hand.kRight) * Constants.ModuleConstants.kMaxModuleAngularSpeedRadiansPerSecond,
+                    False
+                ),
+                self.m_robotDrive
+            )
+        )
+
+    def configureButtonBindings(self):
+        pass
+
+    def getAutonomousCommand(self):
+        """
+        Create config for trajectory
+        """
+        config = TrajectoryConfig(
+            Constants.AutoConstants.kMaxSpeedMetersPerSecond,
+            Constants.AutoConstants.kMaxAccelerationMetersPerSecondSquared
+        ).setKinematics(Constants.DriveConstants.kDriveKinematics)
+
+        """
```

Comment:
By convention, docstrings are only used for describing functions. Normal `#` comments are used for all other comments.